### ISSUE TITLE
E203 erroneously flagging correctly formatted slice definition

### ISSUE THREAD

**Issue by malikoth** [grantmcconnaughey/Flake8Rules#30]
I have a line like this inside a loop in a generator:

```python
yield collection[i : i + 10]
```

`black` insists on formatting it like that, and `flake8` insists on flagging it as having improper whitespace before the colon.  The pep8 discussion on [whitespace](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) shows examples of slices including spaces before the colon as acceptable conventions.

**Comment by malikoth** [comment]
Yeah, this is totally the wrong place to report this.  Sorry, my bad.  <3
